### PR TITLE
US001 View Job Roles

### PR DESCRIPTION
**Pull request by Cormac-F**
- Created JobService, Job, JobController, JobDAO.
- Created endpoint to view job role and job id
- SQL query to create the relevant tables
Screenshot 2023-12-12 at 2 00 30 pm

### CODE REVIEW

**Review by CharlesM988 — APPROVED** [review]
Running fine for me. Can't see any issues.

Screenshot 2023-12-12 at 14 31 52

File: src/main/java/org/kainos/ea/DropwizardApplication.java [hunk 1]
```diff
@@ -30,6 +31,7 @@ protected SwaggerBundleConfiguration getSwaggerBundleConfiguration(DropwizardCon
     public void run(final DropwizardConfiguration configuration,
                     final Environment environment) {
         // TODO: implement application
```
Comment (jkelso662) [inline src/main/java/org/kainos/ea/DropwizardApplication.java]: Would remove TODO now that you have implemented this

**Review by lullabelle — DISMISSED** [review]
Hi
Just wondering why you have created a separate db script than the one in ticket https://github.com/Cormac-F/infinite-harmony-api/pull/15/files? Would you not add this to the db skeleton script to build up a schema?

**Comment by Cormac-F** [comment]
> Hi Just wondering why you have created a separate db script than the one in ticket https://github.com/Cormac-F/infinite-harmony-api/pull/15/files? Would you not add this to the db skeleton script to build up a schema?
The one in the referenced ticket is the entire database that has already been created. The script in this ticket references the tables created for the functionality of this ticket.

**Comment by Cormac-F** [comment]
The unnecessary SQL file has been removed.

**Review by jkelso662 — APPROVED** [review]
Good job! Seems like a solid feature. Shame there are no unit tests! 🙃

**Pull request merged**